### PR TITLE
`go-commander` -> `cobra`

### LINKED ISSUE #867: Evaluate options and standardize our usage of CLI utility
The main app uses go-commander and newer testtool uses something else. Let's standardize.

### PR DESCRIPTION
**Pull request by tamird**
cobra uses `pflags` over `flags`, and `pflags` strictly treats
single-dash flags as shorthands, so all invocations including flags
have grown a leading dash.

Fixes #867.

Moar info: https://github.com/ogier/pflag#command-line-flag-syntax

### CODE REVIEW

**Comment by tschottdorf** [comment]
```
server/cli/accounting.go:23:2: cannot find package "github.com/spf13/cobra" in any of:
	/usr/src/go/src/github.com/spf13/cobra (from $GOROOT)
	/go/src/github.com/spf13/cobra (from $GOPATH)
server/cli/flags.go:23:2: cannot find package "github.com/spf13/pflag" in any of:
	/usr/src/go/src/github.com/spf13/pflag (from $GOROOT)
	/go/src/github.com/spf13/pflag (from $GOPATH)
```

updated GLOCKFILE and did the `make listdeps` thing in `godeps.sh`?

**Comment by petermattis** [comment]
You probably need to update the `GLOCKFILE` and `build/devbase/godeps.sh`.

**Comment by tamird** [comment]
@tschottdorf @petermattis done

**Comment by tschottdorf** [comment]
LGTM

**Comment by MBerhault** [comment]
LGTM

**Comment by petermattis** [comment]
Fixes #867 

LGTM

File: server/cli/cli.go [hunk 1]
```diff
@@ -63,57 +63,59 @@ Output build version information.
...
-		lsZonesCmd,
-		rmZoneCmd,
-		setZoneCmd,
-
-		// Miscellaneous commands.
-		// TODO(pmattis): stats
-		listParamsCmd,
-		versionCmd,
-	},
+var allCmds = []*cobra.Command{
```
Comment (spencerkimball) [inline server/cli/cli.go]: Does this CLI package give us nested commands?

**Comment by spencerkimball** [comment]
LGTM

File: server/cli/cli.go (hunk 1 above)
Comment (tamird) [inline server/cli/cli.go]: yep, see https://github.com/cockroachdb/cockroach/pull/922/files#diff-fd6f83b6931534806d1a150828e05695R118

maybe that should happen on `init()`

**Pull request merged**